python/lsst/obs/pfs/constructNormFlatTask.py

```diff
@@ -95,8 +90,8 @@ def combine(self, cache, struct):
         for expRef in dataRefList:
             exposure = expRef.get('postISRCCD')
             xOffsets.append(exposure.getMetadata().get('sim.slit.xoffset'))
-            fiberTrace = expRef.get('fiberTrace')
-            fts = 
+            fiberTrace = expRef.get('fiberTrace', immediate=True)
+            fts = makeFiberTraceSet(fiberTrace, exposure)
```

Comment:
Other comments have mentioned this.  You need to make sure that the object coming from the butler is the one you want.